config/settings.yml

```diff
@@ -164,6 +164,11 @@ global_navigation:
       visibility:
         - admin
 
+    addresses:
+      path: /contacts
+      visibility:
+        - BRANSTON
+
```

Comment:
Oh my goodness - that was so much simpler than what I was trying to do 🤦 completely forgot we had this bit!

Reply:
Yeah I was looking through the code and I stumbled upon this settings file so I thought, why not give it a try! :)